src/main/java/nextstep/ladder/domain/RandomCondition.java

```diff
@@ -0,0 +1,10 @@
+package nextstep.ladder.domain;
+
+import java.util.Random;
+
+public class RandomCondition implements ConditionStrategy {
+    @Override
+    public boolean randomBoolean() {
+        return new Random().nextBoolean();
+    }
```

Comment:
매번 생성하지 않고 상수로 만들어 보면 좋겠네요.

```suggestion
    private static Random random = new Random();

    @Override
    public boolean randomBoolean() {
        return random.nextBoolean();
}
```